### PR TITLE
[IMP] accounting: update the doc related to internal transfers

### PR DESCRIPTION
**Pull request by Donapi**
task [id:](https://www.odoo.com/web#id=2901915&cids=1&menu_id=5221&model=project.task&view_type=form) 2901915

### CODE REVIEW

File: content/applications/finance/accounting/bank/misc/interbank.rst [hunk 1]
```diff
@@ -1,132 +1,146 @@
...
 
-Register an internal transfer from one bank to another.
-=======================================================
+Register an internal transfer from one bank to another - use case
+=================================================================
 
-We will now transfer 50.000 euros from our **Bank** to our **Bank BE57 0633
-9533 1498** account.
+Let's make an internal transfer for a company that has two bank accounts registered on its database.
+We want to make a transfer of 1.000 USD from Bank A to Bank B account.
```
Comment (jaae-odoo) [inline content/applications/finance/accounting/bank/misc/interbank.rst:25]:
```suggestion
We want to make a transfer of 1.000 USD from Bank A to Bank B.
```
I don't think we can put the word "account " as singular, as there are two accounts (A and B). If we want to put the word "account", we should say 'from Bank A's account to Bank B's account', or 'from Bank A's account to Bank B's one', but it seems simpler just to remove the word "account".

**Review by jaae-odoo — COMMENTED** [review]
Top cette PR ! J'ai mis juste un commentaire pour une phrase que perso je formulerais différemment, mais c'est un détail :)

**Review by jaae-odoo — APPROVED** [review]

File: content/applications/finance/accounting/bank/misc/interbank.rst [hunk 2]
```diff
@@ -1,132 +1,146 @@
...
+   * - **Internal transfer account**
+     -
+     - **$1.000**
 
 .. note::
-   The memo is important if you wish to automatically reconcile (see `Import bank statements and
-   reconcile`_).
+   There are one outstanding payment and one outstanding receipt pending in your two bank account
+   journals, because we did not book yet the bank statement confirming the sending and receiving of
+   the money.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/accounting/bank/misc/interbank.rst:81]: Rephrased to remove the "we" as you suggested but the phrase is not shorter and not sure the turn of phrase is better. Feel free to suggest something else if you don't like it this way :)

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**